docsets/Faker/docset.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Faker",
+    "version": "1.7.1",
+    "archive": "versions/Faker-1.7.1.tgz",
```

Comment:
This can't point to an archive within `specific_versions`. It needs to be an archive of its own, even if it's a duplicate.